action.yml

```
@@ -12,6 +12,8 @@ inputs:
   command:
     description: "The chart-testing command to run"
     required: true
+  dockerArgs:
```

Use snake case to be consistent with chart-releaser-action
```suggestion
  docker_args:
```